doc/conf_ref.rst

```diff
@@ -222,6 +222,19 @@ or :ref:`mirrorlist <mirrorlist-label>` option definition.
     RPM debug scriptlet output level. One of: ``critical``, ``emergency``,
     ``error``, ``warn``, ``info`` or ``debug``. Default is ``info``.
 
+.. _search_enabled_metadata-label:
+
+``search_enabled_metadata``
+    :ref:`boolean <boolean-label>`
+
+    The option is applied for install command if package cannot be found in enabled repositories.
+    Set this to ``True`` for possibility to temporarily enable disabled repositories with
+    :ref:`enabled_metadata=1 <enabled_metadata-label>`. Please be very careful with this option
+    because if dnf is executed with the ``--assumeyes`` or ``-y`` option, or if the ``assumeyes``
+    directive is enabled in ``/etc/dnf.conf``, it enables disabled repositories without prompting
+    for confirmation. This may lead to problems, for example, enabling repositories that you do not
+    want enabled.
```

Comment:
We cannot set the new behavior as default as yum do not set behavior in search-disabled-repos as default. By default it only informs that there is a possibility to use the feature. But for automatic script that use --assumeyes the new behavior is pretty dangerous.